apps/protocol-frontend-e2e/src/support/metamaskMockExt.ts

```diff
@@ -0,0 +1,123 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import { JsonRpcProvider, JsonRpcSigner } from '@ethersproject/providers';
+import '@testing-library/cypress/add-commands';
+import { BigNumber, ethers } from 'ethers';
+import { Resolver } from "@ethersproject/providers";
+
+export class MockExtensionProvider extends ethers.providers.BaseProvider {
```

Comment:
It might be good to add a comment citing the original mock with the permalink

Reply:
Woah! The original mock has been deleted!! Now they're using another interesting library _https://github.com/rsksmart/mock-web3-provider_ . Might explore this for the next steps (#walletConnect)
Will cite the repo which has the original mock still

Reply:
Migrate to this to remove the cookie and address environment variable. 
![Screenshot from 2022-09-14 21-39-53](https://user-images.githubusercontent.com/12705146/190293248-10369029-8608-4502-8579-538c77d6fa50.png)

Reply:
https://github.com/rsksmart/rlogin-sample-apps/blob/main/basic-dapp/cypress/integration/injected_spec.js